File: Ejercicios_En_Clase/promedio/Codificación/Promedio3Notas.java
package promedio3notas;

import java.util.Scanner;

/**
 *
 * @author USUARIO
 */
public class Promedio3Notas {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        
        double n1, n2, n3, promedio;
        
        System.out.print("Ingrese la Nota 1: ");
        n1 = scan.nextDouble();    
        System.out.print("Ingrese la Nota 2: ");
        n2 = scan.nextDouble();
        System.out.print("Ingrese la Nota 3: ");
        n3 = scan.nextDouble();
        
        promedio = (n1+n2+n3)/3;
        
        System.out.println("\n====== Notas Ingresadas ======");
        System.out.println("Nota1: "+n1);
        System.out.println("Nota2: "+n2);
        System.out.println("Nota3: "+n3);
        System.out.println("Su promedio es: "+promedio);
    }
    
}

File: Ejercicios_En_Clase/promedio/Codificación/Promedio3Notas.cpp
#include <iostream>

using namespace std;

int main() {
    double n1, n2, n3, promedio;

    cout << "Ingrese la Nota 1: ";
    cin >> n1;
    
    cout << "Ingrese la Nota 2: ";
    cin >> n2;
    
    cout << "Ingrese la Nota 3: ";
    cin >> n3;

    promedio = (n1 + n2 + n3) / 3.0;

    cout << "Primera nota ingresada: " << n1 << endl;
    cout << "Segunda nota ingresada: " << n2 << endl;
    cout << "Tercera nota ingresada: " << n3 << endl;
    cout << "Su promedio es: " << promedio << endl;

    return 0;
}